validator/example/cmd_test.go

```diff
@@ -11,15 +11,24 @@ func Test_ExecuteCommand(t *testing.T) {
 
 	// Testing cobra commands with default recommended config
 	// default config can also be overrided
-	var vali rules.RuleConfig
-	validationErr := vali.ExecuteRules(cmd)
+	ruleCfg := rules.ValidatorConfig{
+		Options: rules.Options{Verbose: false},
```

Comment:
Too Generic name. Options of what?

Reply:
done! golangci-lint failing

Reply:
@ankithans I think fixing linting rules will be separate issue - good first issue